### PR TITLE
Feature/moz 337 homepage activities

### PR DESCRIPTION
**Pull request by kycapstick**
Added featured activities and active campaign to homepage.
Pulled over logic from campaigns page to check date/status
Added ACF export to ACF folder.

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -206,6 +239,120 @@
...
 			<?php endif; ?>
+			<?php if($hasEvents): ?>
+			<div class="homepage__events">
+				<div class="homepage__events__background"></div>
+				<div class="row homepage__events__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__events__subheader"><?php echo __($fieldValues->featured_events_title)?></h2>
+					</div>
+					<div class="col-md-6 col-sm-12 homepage__events__cta">
+						<a href="/events" class="btn btn--small btn--dark"><?php echo __($fieldValues->featured_events_cta_text); ?></a>
```
Comment (dnaluz) [inline front-page.php]: This doesn't need to be wrapped in __() function.  This should only be used for static text

File: front-page.php [hunk 2]
```diff
@@ -206,6 +239,120 @@
 				</div>
 			</div>
 			<?php endif; ?>
+			<?php if($hasEvents): ?>
+			<div class="homepage__events">
+				<div class="homepage__events__background"></div>
+				<div class="row homepage__events__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__events__subheader"><?php echo __($fieldValues->featured_events_title)?></h2>
```
Comment (dnaluz) [inline front-page.php]: This doesn't need to be wrapped in __() function.  This should only be used for static text

File: front-page.php [hunk 3]
```diff
@@ -35,73 +41,100 @@
...
+							<img src="<?php echo $fieldValues->hero_image['url'] ?>" alt="<?php echo $fieldValues->hero_image['alt'] ?>">
+						</div>
 					</div>
-					<div class="col-md-6 col-sm-12 homepage__events__cta">
-						<a href="/events" class="btn btn--small btn--dark"><?php echo __($fieldValues->featured_events_cta_text); ?></a>
+					<div class="col-md-4 col-md-offset-1">
+						<div class="homepage__content">
+							<h1 class="homepage__hero__title title title--main"><?php echo $fieldValues->hero_title; ?></h1>
+							<p class="homepage__hero__subtitle subtitle"><?php echo $fieldValues->hero_subtitle ?></p>
+							<a href="<?php echo (is_user_logged_in() ? esc_attr($fieldValues->hero_cta_existing) : esc_attr($fieldValues->hero_cta_new)) ?>"class="btn btn--dark btn--small homepage__hero__cta"><?php echo __($fieldValues->hero_cta_text) ?></a>
```
Comment (dnaluz) [inline front-page.php]: This doesn't need to be wrapped in __() function.  This should only be used for static text

File: front-page.php [hunk 4]
```diff
@@ -35,73 +41,100 @@
...
-			<?php if($hasEvents): ?>
-			<div class="homepage__events">
-				<div class="homepage__events__background"></div>
-				<div class="row homepage__events__meta">
-					<div class="col-md-6 col-sm-12">
-						<h2 class="subheader homepage__events__subheader"><?php echo __($fieldValues->featured_events_title)?></h2>
+				<div class="row">
+					<div class="col-md-5 homepage__hero__splash">
+						<div class="homepage__hero__image">
+							<img src="<?php echo $fieldValues->hero_image['url'] ?>" alt="<?php echo $fieldValues->hero_image['alt'] ?>">
```
Comment (dnaluz) [inline front-page.php]: Should check if these keys are set.  Might throw an error if not

File: front-page.php [hunk 5]
```diff
@@ -206,6 +239,120 @@
...
+			<?php endif; ?>
+			<?php if(isset($fieldValues->featured_activities)): ?>
+			<div class="homepage__activities">
+				<div class="homepage__activities__background"></div>
+				<div class="row homepage__activities__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__activities__subheader"><?php echo __($fieldValues->featured_activities_title)?></h2>
+					</div>
+					<div class="col-md-6 col-sm-12 homepage__events__cta">
+						<a href="/activities" class="btn btn--small btn--dark"><?php echo __($fieldValues->featured_activities_cta_text); ?></a>
```
Comment (dnaluz) [inline front-page.php]: Should do a check to see if these fields have values before rendering to screen.

This doesn't need to be wrapped in __() function.  This should only be used for static text

File: front-page.php [hunk 6]
```diff
@@ -206,6 +239,120 @@
...
+					</div>
+				</div>
+			</div>
+			<?php endif; ?>
+			<?php if(isset($fieldValues->featured_activities)): ?>
+			<div class="homepage__activities">
+				<div class="homepage__activities__background"></div>
+				<div class="row homepage__activities__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__activities__subheader"><?php echo __($fieldValues->featured_activities_title)?></h2>
```
Comment (dnaluz) [inline front-page.php]: Should check if this field has a value before rendering to screen

**Comment by dnaluz** [comment]
@kycapstick 

Top corners aren't rounded

Screen Shot 2020-02-20 at 2 12 10 PM

**Comment by kycapstick** [comment]
Great catches @dnaluz! Changes pushed for these, also double checked rest of file to implement recommendations.

**Pull request merged**